schema/Core/Navigation.entityType.php

```diff
@@ -10,6 +10,7 @@
     'description' => ts('Table to store navigation.'),
     'add' => '3.0',
     'label_field' => 'label',
+    'parent_field' => 'parent_id',
```

Comment:
This is not part of the official `entityType` spec. At some point we might add it but for now you can leave this out of the commit and Api4 ought to figure it out automatically. If not add it as an annotation in `Civi/Api4/Navigation.php`.